app/code/Magento/Catalog/Controller/Adminhtml/Category/Add.php

```diff
@@ -6,49 +6,55 @@
  */
 namespace Magento\Catalog\Controller\Adminhtml\Category;
 
-use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Model\View\Result\Page;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Catalog\Controller\Adminhtml\Category;
+use Magento\Framework\App\Action\HttpGetActionInterface;
 
 /**
  * Class Add Category
  *
  * @package Magento\Catalog\Controller\Adminhtml\Category
  */
-class Add extends \Magento\Catalog\Controller\Adminhtml\Category implements HttpGetActionInterface
+class Add extends Category implements HttpGetActionInterface
 {
     /**
-     * Forward factory for result
+     * Common Factory to create any Result object
      *
-     * @var \Magento\Backend\Model\View\Result\ForwardFactory
+     * @var ResultFactory
      */
-    protected $resultForwardFactory;
+    protected $resultFactory;
```

Comment:
Hi,
`$this->resultFactory` already defined as protected property in `\Magento\Framework\App\Action\AbstractAction`, as this class extends it - it's already there, we don't need to add it again.

Reply:
Done, thank you.